draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -317,7 +315,7 @@ The term "origin", the mechanism of deriving an origin from a URI, and the "the
 same" matching algorithm for origins are defined in {{RFC6454}}.
 
 "Safe" HTTP methods include `GET`, `HEAD`, `OPTIONS`, and `TRACE`, as defined
-in Section 4.2.1 of {{RFC7231}}.
+in Section 8.2.1 of {{!HTTPSEM=I-D.ietf-httpbis-semantics}}.
```

Comment:
```suggestion
in Section 9.2.1 of {{HTTPSEM}}.
```

Reply:
I'm not entirely familiar with the reference syntax here, but this looks like a duplicate inline reference definition?